app/controllers/books_controller.rb

```diff
@@ -0,0 +1,9 @@
+class BooksController < ApplicationController
+  
+  def show
+    @book = Book.find(params[:id])
+    @avg = @book.average_rating
+    @max = @book.maximum_rating
+    @min = @book.minimum_rating
+  end
```

Comment:
Your ultimate goal is to only pass one instance variable to the view.  It can definitely get tricky to do this, and you'll learn about some more patterns to accomplish this goal in Mod3. Here though, you could pass @book to the view and from the view call @book.average_rating etc. 

You might have not done this in trying to get logic out of the view, but it is considered okay to call a method from an object in a view. In this sense @book.average_rating is as much logic as @book.title.